Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -101,13 +107,18 @@ static const ReportFeatures reporting_mode_features[] = {
     {0, 0, 0, 0, 23},
 };
 
+// Used for extension calibration data:
+// Last two bytes are the sum of the previous bytes plus 0x55 and 0xaa
+void UpdateCalibrationDataChecksum(std::array<u8, 0x10>& data)
+{
+  const u8 ck1 = std::accumulate(std::begin(data), std::end(data) - 2, (u8)0x55);
```

Comment:
How was 0x55 derived?  If this has a special meaning, it might be good to make this a variable.

Reply:
I do have a comment above the function. It's just a value that Nintendo decided on I guess?

Reply:
`ck1` is fairly nondescript, especially since theres no `ck2` following it. Would you mind using a nicer variable name (such as `checksum`)?

Also:
```suggestion
  const u8 ck1 = std::accumulate(std::begin(data), std::end(data) - 2, static_cast<u8>(0x55));
```

Reply:
I can call it checksum if you want. I figured it was obvious enough considering the small size of the function. And I'll change the u8 to a function-style cast.